README.rst

```diff
@@ -51,6 +51,29 @@ Finally, configure the Django settings, run migrations, and launch the developme
     $ ./manage.py runserver
 
 
+Code Quality
+------------
+
+Install black <https://pypi.org/project/black/> and integrate it with your editor of choice.
+Run flake8 <http://flake8.pycqa.org/en/latest/> to ensure you don't increase the warning count or introduce errors with your commits.
```

Comment:
I don't know whether you have a line-length standard for Restructured Text documents. I'm increasingly using Markdown entirely so I can just run Prettier on it (it became a core format at some point while I was out) but I think there should be a ReST  formatter, too.